docs/config/README.md

```diff
@@ -3814,6 +3814,7 @@ To enable it, set `disabled` to `false` in your configuration file.
 | `fish_indicator`       | `'fsh'`                   | A format string used to represent fish.                      |
 | `zsh_indicator`        | `'zsh'`                   | A format string used to represent zsh.                       |
 | `powershell_indicator` | `'psh'`                   | A format string used to represent powershell.                |
+| `pwsh_indicator`       | `'psh'`                   | A format string used to represent pwsh.                      |
```

Comment:
`None` defaults are usually empty in the table, but perhaps add a comment with an explanation. Now that I am thinking about it, setting the default to `$powershell_indicator` instead of `None` should also work in this case.